packages/flutter/test/material/data_table_test.dart

```diff
@@ -309,6 +312,54 @@ void main() {
     expect(tester.takeException(), isNull);
   });
 
+  testWidgets('DataTable sort indicator orientation test', (WidgetTester tester) async {
```

Comment:
Minor nit:

```suggestion
  testWidgets('DataTable sort indicator orientation', (WidgetTester tester) async {
```

The 'test' here is redundant, as they are all tests 😄 .